guake/tests/test_guake.py

```diff
@@ -1,2 +1,106 @@
-def test_000():
-    pass
+# -*- coding: utf-8 -*-
+# pylint: disable=redefined-outer-name
+
+import json
+import os
+import pathlib
```

Comment:
I recommend using [path.py](https://pypi.org/project/path.py/) instead of pathlib :)

Reply:
I still stuck with `pathlib.Path`, since most of the `Path` object in the code is using `pathlib`, and using `path.py` make some breakage, e.g.
* `Path().exists()` (pathlib) -> `os.path.exists(Path())` (path.py)
*  `with Path().open() as f` (pathlib) -> `with open(Path()) as f` (path.py)

Reply:
ok for me.

But path.py still provides the .exists() function, .open() like pathlib does. It is basically pathlib + mkdirs + convert to str smoothly :)

Reply:
hmmm, so it is caused by fakefs? I'm not sure about this :(